otter/convergence/service.py

```diff
@@ -438,8 +442,11 @@ def buckets_acquired(self, my_buckets):
 
         This is used as the partitioner callback.
         """
-        eff = Effect(GetChildren(CONVERGENCE_DIRTY_DIR)).on(
+        ceff = Effect(GetChildren(CONVERGENCE_DIRTY_DIR)).on(
             partial(self._converge_all, my_buckets))
+        eff = Effect(Func(uuid.uuid1)).on(
+            lambda uid: with_log(ceff, system='converger',
```

Comment:
oh wait, no, I do understand, since we're not using self.log any more. then I guess my same comment applies: please use `otter_service` instead of (or in addition to) `system`